plugins/frontend/backstage-plugin-iframe/src/plugin.ts

```diff
@@ -54,3 +55,18 @@ export const EntityIFrameContent = iframePlugin.provide(
   }),
 );
 
+export const HomePageIFrameCard = homePlugin.provide(
+  createCardExtension<{
+    src: string;
+    title?: string;
```

Comment:
You don't need to give the title prop here or do you use it inside the `./components/HomePageCard` ?

Reply:
Ah I see, you use it, so take this as I didn't say anything

Reply:
I'll remove it!